pyo3-macros-backend/src/pyclass.rs

```diff
@@ -1399,6 +1400,28 @@ impl<'a> PyClassImplsBuilder<'a> {
         }
     }
 
+    fn impl_into_pyclassinit(&self) -> TokenStream {
```

Comment:
@davidhewitt I've tried removing the blanket `impl From<T> for PyClassInitializer<T>` stuff and generating it for each `#[pyclass]` like this.

This has problems because these impls rely on internal details of the module in which they were originally defined.

Maybe it's better to instead generate `impl Into<PyClassInitializer<T>> for T` instead?

Reply:
I don't immediately see what the advantage of implementing `Into` is, but if you see a clear path, we can certainly explore it.

In general, what I'd recommend when you need to rely on crate internals like this, is to put one or more hidden public APIs inside `src/impl_/pyclass.rs`, and call that from the macro generated code.

I actually did this on stream on Friday to build #3744 :)